documentation/docs/02-template-syntax/05-styles-and-classes.md

```diff
@@ -66,6 +66,36 @@ The `-global-` part will be removed when compiled, and the keyframe will then be
 </style>
 ```
 
+## :global
+
+To apply all styles after a certain point to a selector globally, use the `:global` modifier.
+
+```svelte
+<style>
+	:global {
+		div {
+			/* this will apply to every <div> in your application */
+			margin: 0;
+		}
+		p {
+			/* this will apply to every <p> in your application */
+			color: blue;
+		}
+	}
+
+	div :global strong {
+		/* this will apply to all <strong> elements, in any
+			component, that are inside <div> elements belonging
+			to this component */
+		color: goldenrod;
+	}
+</style>
+```
+
+The difference between `:global` and `:global(...)` is that `:global(...)` only makes all styles within its braces global, whereas `:global` makes all styles coming after it global, including those in nested CSS.
+
+If `:global` is preceeded by a descendant combinator, the combinator is removed from the output. That means that `div :global.x` is equivalent to `div.svelte-hash.x`.
```

Comment:
```suggestion
> Some preprocessors will flatten nested selectors, turning the second example above into a `div :global strong` selector. As such this form is also supported, though not recommended.
```

Reply:
Why remove the existing sections? We could adjust them to only have global-with-nesting examples

Reply:
Because we _shouldn't_ be recommending `div :global strong`, at least not as long as `div :global(strong)` is supported. That's just two very similar ways of achieving the same thing, which is something we generally strive to avoid.

Even if there are multiple selectors, I would argue that this...

```css
a :global(b c d) {...}
```

...is much clearer than this:

```css
a :global b c d {...}
```

I really think that from an authoring perspective, we should promote `:global {...}` as the default, and `:global(...)` as a more occasional escape hatch, and `:global selector` not at all other than in the context of 'this happens to work because reasons'.

Reply:
I would be ok with removing the second paragraph though I think it's worthwhile to know how to reason about it, so if there's a way to phrase it while using an example that doesn't promote the "bad" way, shouldn't we keep it?
And the first example doesn't mention anything with regards to how to write it, it provides an important comparison IMO and so I'd like to keep it

Reply:
ok what about this? we replace all the element selectors with class selectors, to make them more abstract and so that we can have multiple selectors after the `:global`:

```css
.a :global {
	/* applies to every `.b .c .d` element, in any component,
	   that are inside an `.a` element in this component
	.b .c .d {...}
}
```

Then we can do this:

```suggestion
> Some preprocessors will flatten nested selectors, turning the second example above into an equivalent `.a :global .b .c .d` selector, where everything after the `:global` is unscoped. As such this form is also supported, but the nested form is recommended instead when writing styles.
```

Reply:
Moving the info into the code example works for me 👍 I'd still like to avoid using preprocessors as the reasoning and instead say "because they are semantically equivalent, this is also allowed.."

Reply:
pushed a tweak, let me know what you think